app/modules/import-export/utils.php

```diff
@@ -115,4 +115,20 @@ public static function get_import_sessions( $should_run_cleanup = false ) {
 
 		return $import_sessions;
 	}
+
+	public static function update_space_between_widgets_values( $space_between_widgets ) {
+		$setting_not_exist = ! isset( $space_between_widgets['size'] );
```

Comment:
maybe use positive 
settting_exist 
?

Reply:
OK, But we will have to negate it in the condition.

Reply:
it wont work otherwise :)

Reply:
We can't always be positive :)

Reply:
LOL